pkg/reconciler/cluster/controller.go

```diff
@@ -145,7 +144,7 @@ type Controller struct {
 	kubeconfig                   clientcmdapi.Config
 	resourcesToSync              []string
 	syncerMode                   SyncerMode
-	syncers                      map[string]*syncer.Syncer
+	syncers                      map[string]func()
```

Comment:
Maybe comment that this holds cancel funcs? Or rename to `syncerCancelFuncs`?

Reply:
Can do